cli_reference/get_started_cli.adoc

```diff
@@ -105,13 +133,23 @@ There are a few ways to provide `osc` a configuration file. Basically when loadi
 .  The `.kubeconfig` file in the current directory
 .  The `.kubeconfig` file inside the `.kube` directory in the user's home: `~/.kube/.kubeconfig`
 
-For details about advanced management of config files, see link:../using_openshift/managing_cli_profiles.html[Managing CLI Profiles].
+You can link:setup_multiple_cli_profiles.html[configure and manage multiple CLI
+profiles] using the `osc config` command.
 
 == Projects
-A link:../architecture/openshift_model.html#project-experimental[project] in OpenShift terms is the package that can contain multiple link:../architecture/openshift_model.html[components] that will compose your logical "link:../v2_changes/applications.html[application]". 
+A link:../dev_guide/projects.html[project] in OpenShift is the package that can
+contain multiple
+link:../architecture/core_objects/openshift_model.html[components] that compose
+your logical "link:../whats_new/applications.html[application]".
 
```

Comment:
[Suggestion]
A project in OpenShift is the package that contains multiple components to make up a logical application.